planet4-gutenberg-blocks.php

```diff
@@ -276,6 +278,7 @@ function set_allowed_block_types( $allowed_block_types, $context ) {
 		PAGE_BLOCK_TYPES,
 		! Features::is_active( 'beta_blocks' ) ? [] : BETA_PAGE_BLOCK_TYPES,
 		! $enform_active ? [] : [ 'planet4-blocks/enform' ],
+		BLOCK_TEMPLATES,
```

Comment:
Do we want them only allowed in pages? I would think that we also want them in Actions at least 🤔